sdk/attestation/attestation/test/utils/recordedClient.ts

```diff
@@ -113,5 +134,19 @@ export function createRecordedAdminClient(
     env.AZURE_CLIENT_ID,
     env.AZURE_CLIENT_SECRET
   );
+
+  // If we're talking to a live server, we should validate the time results,
+  // otherwise we want to skip them.
+  if (options === undefined) {
+    options = {
+      validationOptions: {
+        validateToken: true,
+        validateExpirationTime: isLiveMode() || isRecordMode(),
```

Comment:
maybe just check for `!isPlaybackMode()`

Reply:
Why? These checks are valid whenever we're interacting with the real server. That is true in both live and record mode.

Reply:
Makes sense.